spec/features/menage_users_by_admin.rb

```diff
@@ -38,5 +38,43 @@
     find('ul > li > a', text: "Navigate to the admin panel").click
     expect(page).to have_current_path(admin_path)
   end
+
+  scenario 'admin can add new user' do
+    visit admin_users_path
+    expect{ page.to_not have_content("new_user@o2.pl")
+            page.to_not have_content('33')
+            page.to_not have_content("female")
+            page.to_not have_content("Programing")
+            page.to_not have_content("work") }
+    create_user
+    expect{ page.to have_content("new_user@o2.pl")
```

Comment:
There is one downside to use just one expect. If it fails you don't know where exactly the error is. If you split every single one to: 
```
expect(page).to have_content("new_user@o2.pl")
````
you will know exactly which line fails.